### PR TITLE
Global Replace (% hightlight xyz %} with ```xyz

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this PR

Removes all old {% highlight %} syntax with ```zyx format.

This will allow us to proceed with #3432 

## Affected URLs

- Many

### CODE REVIEW

File: guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md [hunk 1]
```diff
@@ -333,7 +333,7 @@ Configuration that is passed to the Slider widget.
 **Aliases**: `[range]`
 
 **Usage example**:
-{%highlight javascript%}
+```javascript
```
Comment (jeff-matthews) [inline guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md]:
```suggestion
```html
```

File: guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md [hunk 2]
```diff
@@ -358,10 +358,10 @@ Configuration for the Resizable widget.
 
 **Usage example**:
 
-{%highlight javascript%}
+```javascript
```
Comment (jeff-matthews) [inline guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md]:
```suggestion
```html
```

File: guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md [hunk 3]
```diff
@@ -375,7 +375,7 @@ Component's name by which to perform a lookup in the registry.
 **Aliases**: `[ko-scope]`, `<scope>`
 
 **Usage example**:
-{%highlight javascript%}
+```javascript
```
Comment (jeff-matthews) [inline guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md]:
```suggestion
```html
```

File: guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md [hunk 4]
```diff
@@ -396,9 +396,9 @@ The `staticChecked` binding implements the behavior similar to the standard [`ch
 
 **Aliases:** `[staticChecked]`
 
-{%highlight javascript%}
+```javascript
```
Comment (jeff-matthews) [inline guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md]:
```suggestion
```html
```

File: guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md [hunk 5]
```diff
@@ -413,9 +413,9 @@ Configuration for the `template` binding. If the provided value is a string, it
 **Aliases:** `[render]`, `<render>`
 
 **Usage example**:
-{%highlight javascript%}
+```javascript
```
Comment (jeff-matthews) [inline guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md]:
```suggestion
```html
```

File: guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md [hunk 6]
```diff
@@ -484,7 +484,7 @@ Binding's configuration that may include the following options:
 
 Adding the tooltip binding as an attribute:
 
-{%highlight javascript%}
+```javascript
```
Comment (jeff-matthews) [inline guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md]:
```suggestion
```html
```

File: guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md [hunk 7]
```diff
@@ -494,11 +494,11 @@ Adding the tooltip binding as an attribute:
  "> Tooltip data </div>
 
 <div data-tooltip-trigger="trigger"/>
-{%endhighlight%}
+```
 
 Adding the tooltip binding as a node:
 
-{%highlight javascript%}
+```javascript
```
Comment (jeff-matthews) [inline guides/v2.1/ui_comp_guide/concepts/knockout-bindings.md]:
```suggestion
```html
```

File: guides/v2.1/coding-standards/code-standard-php.md [hunk 8]
```diff
@@ -27,10 +27,10 @@ This includes references to:
 
 Examples:
 
-{% highlight php startinline=true %}
+```php
```
Comment (jeff-matthews) [inline guides/v2.1/coding-standards/code-standard-php.md]: have you verified that this renders properly without the additional inline syntax?

> php?start_inline=1

there are _many_ php code blocks in this PR that could be affected if this is required.

File: guides/v2.1/coding-standards/technical-guidelines.md [hunk 9]
```diff
@@ -193,7 +193,7 @@ class Config
     </tr>
     <tr>
         <td>
-{% highlight php %}
+```php
```
Comment (keharper) [inline guides/v2.1/coding-standards/technical-guidelines.md]: Will the code gates render correctly within an HTML table?

File: guides/v2.1/comp-mgr/trouble/cman/tshoot_backup.md [hunk 10]
```diff
@@ -17,16 +17,16 @@ A backup can fail for any of the following reasons:
 
 ## Backup disabled
 
-If the Magento backup functionality does not start or displays the following message, you need to enable the feature prior to backing up. 
+If the Magento backup functionality does not start or displays the following message, you need to enable the feature prior to backing up.
 
-```
+```bash
```
Comment (keharper) [inline guides/v2.1/comp-mgr/trouble/cman/tshoot_backup.md]: change to terminal

File: guides/v2.1/design-styleguide/iconography/iconography.md [hunk 11]
```diff
@@ -94,14 +94,14 @@ Modern versions of assistive technologies will announce CSS generated content, a
 
 **Code sample:**
 
-{% highlight php %}
+```php
```
Comment (keharper) [inline guides/v2.1/design-styleguide/iconography/iconography.md]: Change to html?

File: guides/v2.1/extension-dev-guide/build/enable-module.md [hunk 12]
```diff
@@ -12,7 +12,7 @@ After you have built the component and are ready to enable it in your Magento en
 1. Disable the {% glossarytooltip 0bc9c8bc-de1a-4a06-9c99-a89a29c30645 %}cache{% endglossarytooltip %} under `System->Cache Management`.
 1. Enter the following in the command line:
 
-   ``` bash
+   ```bash
```
Comment (keharper) [inline guides/v2.1/extension-dev-guide/build/enable-module.md]: Wrap each command with the bash code blocks separately.

File: guides/v2.1/coding-standards/code-standard-php.md (hunk 8 above)
Comment (dobooth) [reply guides/v2.1/coding-standards/code-standard-php.md]: I will check them. Seems to work when all spacing/indentation rules are followed.

File: guides/v2.1/coding-standards/technical-guidelines.md (hunk 9 above)
Comment (dobooth) [reply guides/v2.1/coding-standards/technical-guidelines.md]: It seems to with proper spacing/indentation. I will check all the PHPs.

File: guides/v2.2/comp-mgr/extens-man/extensman-update.md [hunk 13]
```diff
@@ -27,7 +27,7 @@ If the update is successful, a page similar to the following displays.
 
 Messages similar to the following display in the Console Log:
 
-{% highlight xml %}
+```xml
```
Comment (keharper) [inline guides/v2.2/comp-mgr/extens-man/extensman-update.md]: Change to terminal

File: guides/v2.2/comp-mgr/extens-man/extensman-new-purchase.md [hunk 14]
```diff
@@ -23,7 +23,7 @@ If the list of purchases is correct, click **Install**.
 
 Messages display in the Console Log as your new purchases are installed. Following is a sample of some of these messages:
 
-{% highlight xml %}
+```xml
```
Comment (keharper) [inline guides/v2.2/comp-mgr/extens-man/extensman-new-purchase.md]: change to terminal

File: guides/v2.1/comp-mgr/trouble/cman/tshoot_backup.md (hunk 10 above)
Comment (dobooth) [reply guides/v2.1/comp-mgr/trouble/cman/tshoot_backup.md]: Done.

File: guides/v2.1/comp-mgr/module-man/compman-new-purchase.md [hunk 15]
```diff
@@ -22,7 +22,7 @@ If the list of purchases is correct, click **Install**.
 
 Messages display in the Console Log as your new purchases are installed. Following is a sample of some of these messages:
 
-{% highlight xml %}
+```xml
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.1/comp-mgr/module-man/compman-new-purchase.md]: Done.

File: guides/v2.2/comp-mgr/extens-man/extensman-new-purchase.md (hunk 14 above)
Comment (dobooth) [reply guides/v2.2/comp-mgr/extens-man/extensman-new-purchase.md]: Done.

File: guides/v2.1/design-styleguide/iconography/iconography.md (hunk 11 above)
Comment (dobooth) [reply guides/v2.1/design-styleguide/iconography/iconography.md]: Got it.

File: guides/v2.1/extension-dev-guide/build/enable-module.md (hunk 12 above)
Comment (dobooth) [reply guides/v2.1/extension-dev-guide/build/enable-module.md]: Thanks.

File: guides/v2.2/comp-mgr/extens-man/extensman-update.md (hunk 13 above)
Comment (dobooth) [reply guides/v2.2/comp-mgr/extens-man/extensman-update.md]: Got it.

**Comment by dobooth** [comment]
running tests

**Comment by dshevtsov** [comment]
The PR breaks formatting within HTML elements like HTML tables.

**Pull request merged**